app/frontend/src/components/base/BaseDisclaimer.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <h3 class="mb-5">Disclaimer and statement of responsibility for Form Designers:</h3>
+    <ul>
+      <li>
+        It is your responsibility to comply with Privacy laws governing
+        the collection, use and disclosure of personally identifiable
+        information.
+      </li>
+      <li>
+        Access to this form designer tool does not inherently grant
+        permission to collect, use or disclose any personally identifiable
+        information.
+      </li>
+      <li>
+        It is your responsibility to obtain consent to collect information
+        as required by law.
+      </li>
+      <li>
+        Before publishing or distributing your form you are required to
+        discuss the intention of the form with your Ministry Privacy
+        Officer and to complete assessments as required.
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BaseDisclaimer'
+};
```

Comment:
Wouldn't consider this a Base component, it's specific to the Common Forms business case and not a generic web component. Only ever going to be used in 1 or 2 specific places. Would put in /designer instead.

Reply:
To add to this - since this component has no actual JS code in it, consider changing this into a functional SFC instead.
Ref: https://vuejs.org/v2/guide/render-function.html#Functional-Components

Reply:
done. good call